LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -106,29 +106,36 @@ public async Task<Twin> GetTwinAsync(CancellationToken cancellationToken = defau
             }
         }
 
-        public async Task<bool> UpdateReportedPropertiesAsync(TwinCollection reportedProperties)
+        public async Task<bool> UpdateReportedPropertiesAsync(TwinCollection reportedProperties, CancellationToken cancellationToken)
         {
+            CancellationTokenSource cts = default;
             try
             {
-                this.deviceClient.OperationTimeoutInMilliseconds = 120000;
+                if (cancellationToken == default)
+                {
+                    cts = new CancellationTokenSource(LoRaDevice.TwinUpdateTimeout);
```

Comment:
as mentioned above, let's move the default into this class.

Reply:
Done in f8eb224